Sources/CardPayments/Models/Card.swift

```diff
@@ -23,6 +23,7 @@ public struct Card: Encodable {
 
     //// :nodoc: This is exposed for convenience in copying card contents to UpdateSetupTokenQuery
     /// The expiration year and month, in ISO-8601 `YYYY-MM` date format.
+    @_documentation(visibility: private)
     public var expiry: String {
```

Comment:
q: Can we remove this property instead? It shouldn't be difficult to move the `"\(expirationYear)-\(expirationMonth)"` string to the demo app.

Reply:
I prefer this since it is a good compromise of less error for us in future in demo app and hiding implementation from public interface.

Reply:
Looks like this is [not public on Android](https://github.com/paypal/paypal-android/blob/main/CardPayments/src/main/java/com/paypal/android/cardpayments/Card.kt). I agree that merchants should not need access to this.

Reply:
I don't actually use this field in Demo app, we have a different `BasicCard` struct. I just have to modify `UpdateSetupToken` encoding.

Reply:
I'll put this in separate PR